internal/installer/elasticagent_tar_macos.go

```diff
@@ -56,7 +56,7 @@ func (i *elasticAgentTARDarwinPackage) AddFiles(ctx context.Context, files []str
 func (i *elasticAgentTARDarwinPackage) Inspect() (deploy.ServiceOperatorManifest, error) {
 	return deploy.ServiceOperatorManifest{
 		WorkDir:    i.metadata.AgentPath,
-		CommitFile: "/elastic-agent/.elastic-agent.active.commit",
+		CommitFile: "elastic-agent/.elastic-agent.active.commit",
```

Comment:
I think we should update the `AgentPath` in [L38](https://github.com/elastic/e2e-testing/pull/2626/files#diff-b9eda3d8d7bfbd3b3f3a8c44c7e06743d03ad92a62fe2543eaba3e80d8e34efdR38)
```suggestion
		CommitFile: "/Library/Elastic/Agent/.elastic-agent.active.commit",
```